test/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+###################
+## Find packages ##
+###################
+
+find_package(GTest)
+
+
+###########
+## Build ##
+###########
+
+if(GTEST_FOUND)
```

Comment:
We _could_ add a warning here if gtest is not found. This CMakeLists should only be configured/built if we explicitly want unit tests.